scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferWithTwoPhaseCommitTransactionProcessor.java

```diff
@@ -29,7 +30,8 @@ public class TransferWithTwoPhaseCommitTransactionProcessor extends TimeBasedPro
 
   public TransferWithTwoPhaseCommitTransactionProcessor(Config config) {
     super(config);
-    this.manager = TransferCommon.getTwoPhaseCommitTransactionManager(config);
+    manager1 = TransferCommon.getTwoPhaseCommitTransactionManager1(config);
+    manager2 = TransferCommon.getTwoPhaseCommitTransactionManager2(config);
```

Comment:
Also, modified the existing transfer workload (without retry) to use different configurations for the transaction managers.